pyproject.toml

```diff
@@ -8,9 +8,9 @@ packages = [{ include = "fastapi_keycloak_middleware" }]
 
 [tool.poetry.dependencies]
 python = "^3.10"
-fastapi = ">=0.73.0"
+fastapi = ">=0.110.0"
```

Comment:
As this library should be used in other projects which may have other dependencies, we want to specify the dependencies as much as possible. I don't consider this a task of this library to educate people on keeping their FastAPI version up to date, rather I'd like to specify the minimum compatible version. May I therefore ask you to revert this change?

Reply:
That's okay, because my current pain is caused by python-jose. However, reverting it in the pyproject.toml as is will not prevent poetry from raising the version to latest in the poetry.lock when updating the dependencies (the current main is already on fastapi 0.101.0).

Reply:
See [7dbdf22](https://github.com/waza-ari/fastapi-keycloak-middleware/pull/29/commits/7dbdf22ead1ea3b1689ca666c3c46f9eb0d3e69d)

Reply:
Luckily the lock file is not taken into account at all when this library is being used as dependency. We do require FastAPI at the moment, although it's really only used for type hinting. I don't want to restrict the users regarding the FastAPI version they can use the library with. Thanks for the change.